webapp/channels/src/components/channel_notifications_modal/__snapshots__/channel_notifications_modal.test.tsx.snap

```diff
@@ -137,6 +142,7 @@ exports[`components/channel_notifications_modal/ChannelNotificationsModal should
             channelAutoFollowThreads="off"
             expand={false}
             ignoreChannelMentions="off"
+            isGM={false}
```

Comment:
Should we add snapshots for GM `true` case?

Reply:
I added several more tests all around the PR. I haven't added any new e2e tests, but I think they are not needed.

Either way, if you think there is any particular behavior that has to be added to the e2e tests, let me know, and I will look into it.

Reply:
LGTM - was looking for covering the case for the settings change in the GM which I see you've covered. Also, thanks for the additional mentions tests in the server 💯